src/main/java/org/zalando/fahrschein/ClientHttpRequestFactoryHelper.java

```diff
@@ -0,0 +1,34 @@
+package org.zalando.fahrschein;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.http.client.ClientHttpRequestFactory;
+import org.springframework.http.client.HttpComponentsClientHttpRequestFactory;
+
+public class ClientHttpRequestFactoryHelper {
```

Comment:
I don't get that class and wrapping. 😟 
And it's also only used in tests but placed in production code.